code/src/components/TodoList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { TodoInput } from './TodoInput.js'
+import { TodoItem } from './TodoItem.js'
+import { TodoSummary } from './TodoSummary.js'
+import { ClearAllButton } from 'lib/ClearAllButton.js'
+import styled from 'styled-components/macro'
+
+export const TodoList = () => {
+  const list = useSelector((store) => store.todos.list)
+
+  return (
+    <TodoListSection>
+      <TodoInput />
+      {
+        list.items.map((item, index) => (
+          <TodoItem itemIndex={index}></TodoItem>
```

Comment:
Don't forget to add a unique key when you're mapping. Since this element doesn't have any children you can also make it self closing: `<TodoItem />`